keystorage/hashicorp/src/testFixtures/java/tech/pegasys/signers/hashicorp/dsl/HashicorpVaultCommands.java

```diff
@@ -39,24 +36,20 @@ public HashicorpVaultCommands(final String vaultUrl) {
 
   public String[] enableSecretEngineCommand(final String vaultRootPath) {
     return new String[] {
-      "vault", "secrets", "enable", "-address=" + vaultUrl, "-path=" + vaultRootPath, "kv-v2",
+      vaultCommand, "secrets", "enable", "-address=" + vaultUrl, "-path=" + vaultRootPath, "kv-v2",
     };
   }
 
   public String[] unseal(final String unsealKey) {
     return new String[] {
-      "vault", "operator", "unseal", "-address=" + vaultUrl, "-format=json", unsealKey
+      vaultCommand, "operator", "unseal", "-address=" + vaultUrl, "-format=json", unsealKey
     };
   }
 
   public String[] putSecretCommand(final String key, final String value, final String path) {
     final String paramString = String.format("%s=%s", key, value);
     return new String[] {
-      "vault", "kv", "put", "-address=" + vaultUrl, path, paramString,
+      vaultCommand, "kv", "put", "-address=" + vaultUrl, path, paramString,
     };
   }
```

Comment:
loginCommand is no longer needed?

Reply:
yes....we are now using environment variable VAULT_TOKEN to perform authorization instead of login command. The login command creates a token at home folder and our parallel test execution was causing intermittent failures due to it (it worked cleanly in docker environment though).